### ISSUE TITLE
Lodestar: Content Options: Display Categories not functional

### ISSUE THREAD

**Issue by KokkieH** [Automattic/themes#696]
## Steps to replicate

1. Activate Lodestar and make sure some posts with tags and categories are published.
2. Open the Customizer and open Content Options
3. View the Posts page and try toggling the Display Tags and Display Categories on and off.

## Result

Tags disappear and reappear on the posts as expected, but toggling the Display Categories option has no effect.

This appears to work fine on the theme demo site, but I'm able to replicate it on a test site. However, on my test site the category doesn't display no matter what I do, while on the site of the user who reported this, the category does display and there's no way to get rid of it.

I'm unable to figure out where this inconsistency is coming from.

Additionally on the theme demo site the category appears at the top with the date and author byline on multi-author sites:

![Customize__News_–_Lodestar](https://user-images.githubusercontent.com/11873759/54902433-109fe300-4ee2-11e9-8c17-7e4e84a68ffd.png)

But on the user's site they appear at the bottom, with Like/sharing buttons positioned between the categories and tags:

![Greg_Osborne_–_Copywriter](https://user-images.githubusercontent.com/11873759/54902499-41801800-4ee2-11e9-8b2d-e382146f0025.png)
![Greg_Osborne_–_Copywriter](https://user-images.githubusercontent.com/11873759/54902522-565cab80-4ee2-11e9-828f-bdd2911b3dcb.png)

I'm not able to replicate this on a test site, as categories just aren't appearing for me at all.

## Expected

For the Display Categories option to actually affect whether or not categories display on posts.

* User report: https://en.forums.wordpress.com/topic/lodestar-does-not-support-display-categories/

**Comment by alaczek** [comment]
Yep, I can confirm hiding categories does not work. Can't get it to work on my test site as well, yet seems to be working on the demo, as you said. Needs more digging.

> But on the user's site they appear at the bottom, with Like/sharing buttons positioned between the categories and tags:
It looks like the customer added the "Posted in X" manually to the content of their posts.

**Comment by KokkieH** [comment]
> It looks like the customer added the "Posted in X" manually to the content of their posts.

Ah, clever customer. I totally missed that.

**Comment by gemmagarner** [comment]
Another report in: https://en.forums.wordpress.com/topic/lodestar-theme-categories-not-showing-up-on-posts/

Categories not being displayed on their site and I've been able to replicate this issue on my test site.

**Comment by arunsathiya** [comment]
Categories not appearing on the posts; another report - 2284989-zen.

**Comment by hacchism** [comment]
Another report - 3094778-zen

**Comment by bsanevans** [comment]
Another report - 3146882-zen

**Comment by abdul5ashraf** [comment]
An important note was found by @Rashed Ripon

>*The issue/bug is actually made intentionally in the theme. I checked the code - https://prnt.sc/tiw57c. The categories will show on the posts if you use a static homepage instead of the default "latest posts" settings. We could've helped if he has a business plan. The user replied and little bit disturbed, asking for an immediate solution.*

I have tested the website related to 3146882-zen. I found that the issue is resolved when changing the Homepage Setting from **Your Latest Posts** to **A static Page**.

**Comment by Robertght** [comment]
Another case here: 4784536-zen
